karaf-features/pom.xml

```diff
@@ -92,6 +92,7 @@
                                 -->
                                 <feature>oce-features-score</feature>
                                 <feature>oce-features-shell</feature>
+                                <!-- TODO: Should I add any of the redundancy modules here? !-->
```

Comment:
Any new feature that can be added here should be added here. They will work so long as they don't import any services from the OSGi registry in the Blueprint.